docs/execution-extras.rst

```diff
@@ -10,6 +10,7 @@ for you are most likely:
 - IBM's :func:`get_all_job_ids<mpqp.execution.connection.ibm_connection.get_all_job_ids>`;
 - Eviden's :func:`get_all_job_ids<mpqp.execution.connection.qlm_connection.get_all_job_ids>`;
 - AWS's :func:`get_all_task_ids<mpqp.execution.connection.aws_connection.get_all_task_ids>`;
+- Google's :func:`get_all_task_ids<mpqp.execution.connection.google_connection.get_all_task_ids>`;
```

Comment:
I think for cirq they are called jobs, not tasks right ?  So we should name it get_all_job_ids ?
At least for Ionq, it is a Job, but maybe it depends on each provider's API ?